### ISSUE TITLE
Seemingly no input for Bomberman Party Edition

### ISSUE THREAD

**Issue by Raginhari7** [stenzek/duckstation#2114]
Slus-01189 Bomberman Party Edition (usa)

Unable to get a response from the game/and/or controller.  It just cycles back and forth from intro video to title screen.   I have done a search for bomberman issues, but this doesn't seem to match any of them.
Not all that sure if it's a controller issue or not, or just something going on with the rom.  The controllers work fine in other games, so it seems specific to this rom, but controls work fine when I load it up on ePSXe, though the game itself runs like complete hot garbage on the emu.  It seems like it runs really well on DS, it just won't respond. :(

**Controller Troubleshoot Report**
No drivers, just plug and play usb converters for old PSX controllers
Using Dinput back end, but the controllers work fine for other games.

**Steps to reproduce**
As far as i can tell, if my controllers and the rom work otherwise, anyone else who tries to play this rom should have the same experience as I.  Unless there is some specific setting I need enabled for this particular rom.

Enhancements are off
I7 9700, GTX 1650 Super
GPU Renderer - D3D11
Volume DS Qt 5.15.0

Thanks for any help. Much love.

**Comment by foxsevent** [comment]
Game works fine here so the issue lies with your setup.
Recheck your controller settings, if it works with other games like you said than there's no reason for it not to work here as well.

**Comment by stenzek** [comment]
Do you have automatic analog mode enabled, and does this game support dualshock? It's listed as supporting it, but maybe it doesn't.

**Comment by foxsevent** [comment]
> Do you have automatic analog mode enabled, and does this game support dualshock? It's listed as supporting it, but maybe it doesn't.

Sorry could've mentioned and yes it does support analog.

**Comment by Raginhari7** [comment]
It says it supports vibration, but not analogue. according to the back of the case.  However, I tested multiple different variables.  Even though my controls seem to be working fine in other games, as well as on this rom on ePSXe, I still figured I would play around with it.  
So I mapped my shitty logitech controller that works with everything, and I mapped straight up keyboard keys.  Figured I would try these, as I'm using USB converters for original PSX controllers (one digital, and one dualshock) and suspected that something in the converter might not be playing nice with this particular setup.  This also did nothing.
I've tried all the different backend options (Xinput, Dinput and SDL), selected both digital and analog from the controller type drop down menu, and have made sure all enhancements were off, and anolog/digital button disengaged.  After each change I would close and reopen the emu, just incase (I'm anal).
I also had multitap on port one engaged, because I wanna play this 3 player with my roomies.  So in the event that that was somehow messing with things, I tried turning that off too.  Still nothing.
Admittedly new to DS, as of a couple days ago.  Wondering if there is another place I should be adjusting settings that could be effecting input?  I'm gonna try to find another copy of the rom perhaps, as Fox doesn't seem to be having the same issue.  But I know the rom atleast works, all be it terribly, on ePSXe.   On there the sound drops out constantly and there are audible clicks and pops, but the controls atleast register.
Thanks again for the replies.
